### PR TITLE
Feature/login form

### PR DESCRIPTION
**Pull request by Picroc**
Night is great time to search for solutions

It is finally working, you can spam me with security codes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,17 +9,25 @@
...
+  "dependencies": {
+    "telegram-api-js": "^1.2.9"
+  },
   "devDependencies": {
     "@babel/cli": "^7.6.4",
     "@babel/core": "^7.6.4",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/polyfill": "^7.6.0",
     "@babel/preset-env": "^7.6.3",
+    "angular": "^1.7.8",
```
Comment (Picroc) [inline package.json]: Oops this should be deleted. I'll do after))

File: src/App.js [hunk 2]
```diff
@@ -1,6 +1,33 @@
 import Login from './templates/login-form';
 import './assets/fonts.css';
 
+telegramApi.setConfig({
+    app: {
+        id: 1166576, /* App ID */
+        hash: '99db6db0082e27973ee4357e4637aadc', /* App hash */
```
Comment (Picroc) [inline src/App.js]: Also I should delete this from code. Maybe switch to private repo?

File: src/App.js [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
+                id: 2, /* DC ID */
+                host: '149.154.167.50',
+                port: 443
+            }
+        ]
+    }
+});
+
+telegramApi.getUserInfo().then(res => { console.log("User info:", res); });
+telegramApi.sendCode('+79821759743');
```
Comment (Picroc) [inline src/App.js]: And here is the magic. Noice

File: package.json [hunk 4]
```diff
@@ -9,17 +9,25 @@
   },
   "author": "Picroc",
   "license": "ISC",
-  "dependencies": {},
+  "dependencies": {
+    "telegram-api-js": "^1.2.9"
```
[the remark this answers is not in the document]
Comment (pivaszbs) [reply package.json]: My bad, in minify version and gzipped it's 57.3 kb, but it's also not quite good

[the remark this answers is not in the document]
Comment (Picroc) [reply package.json]: Either this or using tdl - I guess we can't do better

**Pull request merged**